components/index.js

```diff
@@ -3,11 +3,19 @@ import ControlledModal from './ControlledModal';
 import FooterTagsList from './FooterTagsList';
 import ControlledDialog from './ControlledDialog';
 import ScrollUpButton from './ScrollUpButton';
+import ComponentVier from './ComponentViewer';
+import HeaderSSR from './HeaderSSR';
+import FooterSSR from './FooterSSR';
+import NavigationMenu from './NavigationMenu';
 
 export {
   ControlledSnackBar,
   ControlledModal,
   ControlledDialog,
   FooterTagsList,
   ScrollUpButton,
+  ComponentVier,
+  HeaderSSR,
+  FooterSSR,
+  NavigationMenu,
```

Comment:
```suggestion
```
As this component is only used in the Header and if it is placed within the SubComponents folder within the Header, this export does not make much sense.